app/forms/idv/doc_pii_form.rb

```diff
@@ -63,6 +66,10 @@ def error_count
       [name_valid?, dob_valid?, state_valid?].count(&:blank?)
     end
 
+    def zipcode_valid?
+      zipcode.nil? || zipcode.is_a?(String)
```

Comment:
WDYT of a length/regex check? 5-9 digits, optional dash?
```suggestion
      zipcode.nil? || zipcode =~ /\A\d{5}(-?\d{4})?\Z/
```

Reply:
I was not confident or familiar in the formats we may get from production proofing vendors, so I wanted to be relatively cautious

Reply:
ok! maybe we do validation on just the YAML parsed versions instead of from here then

Reply:
The receiving/parsing of the data happens right before the validation, so I think this is may be the best place to do it. I suspect we want to validate all of the time as well since we do the string slicing for all environments.